### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by gurbuzertunga**

### CODE REVIEW

**Comment by gurbuzertunga** [comment]
@dasileker can you review my code?

**Comment by lirad** [comment]
Testing reviewing

**Comment by gurbuzertunga** [comment]
@DcRonan can you review my code?

File: index.html [hunk 1]
```diff
@@ -1,8 +1,37 @@
 <html>
+    <link rel="stylesheet" href="styles.css"> 
     <head>
```
Comment (lirad) [inline index.html]: Add  a header tag and check this article about the head tag https://www.w3schools.com/tags/tag_head.asp

File: index.html [hunk 2]
```diff
@@ -3,14 +3,23 @@
...
             <img src="img/avatar-icon.png" alt="Avatar Icon" id="avatar">
         </nav>
     </head>
     <body>
-        
+        <iframe width="640" height="360" src="https://www.youtube.com/embed/dQw4w9WgXcQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+        </iframe>
+        <h3>Rick Astley - Never Gonna Give You Up (Video)
+       </h3>
+       <h4 color="gray">7.578.549 views</h4>
```
Comment (DcRonan) [inline index.html]: The full stops should be commas.

**Review by DcRonan — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+                <nav>
+            <div id="logo">
+                <img src="img/youtube-logo.png" alt="Youtube Company logo">
+            </div>
+            <input type="text" placeholder="Search your music"></input>
+            <button type="submit"></button>
+            <img src="img/notification-icon.png" alt="Notification Icon">
+            <img src="img/upload-icon.png" alt="Upload Icon">
+            <img src="img/avatar-icon.png" alt="Avatar Icon">
+        </nav>
```
Comment (DcRonan) [inline index.html]: The code above including the images, div, input and button should be in the body tag and not the head tag.

**Review by DcRonan — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -1,8 +1,37 @@
...
     <body>
-        
+        <div class="video">
+            <iframe style="width: 800; height: 400px;" src="https://www.youtube.com/embed/dQw4w9WgXcQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+            </iframe>
+        </div>
+        <div class="subvideo">
+            <h3>Rick Astley - Never Gonna Give You Up (Video)
+            </h3>
+            <div class="weirdbuttons">
```
Comment (lirad) [inline index.html]: lol, funny name to a class

File: index.html [hunk 5]
```diff
@@ -1,8 +1,37 @@
...
+                <div id="viewcount"><h4 color="gray">7.578.549 views</h4></div>
```
Comment (lirad) [inline index.html]: as suggestion use either camelCase or a dash separated naming style ex: view-count or viewCount

**Comment by gurbuzertunga** [comment]
@oracleot can you review my code?

**Comment by lirad** [comment]
Try to normalize the CSS first so it will be compatible between browsers: https://meyerweb.com/eric/tools/css/reset/

**Comment by oracleot** [comment]
Try to use more of semantic HTML elements:
https://www.w3schools.com/html/html5_semantic_elements.asp

**Review by RICKCOYL — APPROVED** [review]

**Pull request merged**